rllib/core/models/tf/encoder.py

```diff
@@ -155,7 +155,7 @@ def _forward(self, inputs: NestedDict, **kwargs) -> NestedDict:
         return NestedDict(
             {
                 ENCODER_OUT: self.net(inputs[SampleBatch.OBS]),
-                STATE_OUT: None,  # inputs[STATE_IN],
+                # STATE_OUT: None,  # inputs[STATE_IN],
```

Comment:
removing? if not we should at least leave some comment on why this is commented.

Reply:
Ah, sorry, this is a leftover from experimentation. I'll undo this ...

Reply:
fixed